packages/wallet/src/components/kyc/VerifyIdentityForm.tsx

```diff
@@ -0,0 +1,110 @@
+import { Button } from '@/ui/Button'
+import { FieldError } from '@/ui/forms/FieldError'
+import { FileUpload } from '@/ui/forms/FileUpload'
+import { Form, useZodForm } from '@/ui/forms/Form'
+import { z } from 'zod'
+
+// mock data, the list will come from Rapyd
+const idTypes = [
+  {
+    id: '1',
+    type: 'id',
+    label: 'ID'
+  },
+  {
+    id: '2',
+    type: 'passport',
+    label: 'Passport'
+  },
+  {
+    id: '3',
+    type: 'driverLicence',
+    label: "Driver's licence"
+  }
+]
+
+const verifyIdentitySchema = z
+  .object({
+    idType: z.string({ invalid_type_error: 'Please select an ID Type' }),
+    frontSideID: z.any(),
```

Comment:
I checked many suggestions of file verification and this seems a common validation for it, to use any and add refine for required